node/src/GlideClient.ts

```diff
@@ -742,7 +743,27 @@ export class GlideClient extends BaseClient {
      * console.log(result); // Output: "key12" - "key12" is a random existing key name from the currently selected database.
      * ```
      */
-    public randomKey(): Promise<string | null> {
+    public async randomKey(): Promise<string | null> {
         return this.createWritePromise(createRandomKey());
     }
+
+    /**
+     * Flushes all the previously watched keys for a transaction. Executing a transaction will
```

Comment:
I think the second phrase should be moved/copied to `watch`.

Reply:
Please, review existing doc for `exec` and transaction. Maybe we should update them as well.

Reply:
I will copy the second sentence to `watch`. I propose to add this link (`https://valkey.io/topics/transactions/`) to both commands:

`+  See https://valkey.io/commands/unwatch/ and https://valkey.io/topics/transactions/ for more details.`